web/src/api/routes.ts

```diff
@@ -1,19 +1,29 @@
 import axios from 'axios';
 import { buildExportQuery } from '../model/export-query';
 import { FlowQuery, MetricFunction } from '../model/flow-query';
-import { Record } from './ipfix';
-import { calculateMatrixTotals, parseStream, StreamResult, TopologyMetrics } from './loki';
+import {
+  AggregatedQueryResponse,
+  calculateMatrixTotals,
+  parseStream,
+  RecordsResult,
+  StreamResult,
+  TopologyMetrics
+} from './loki';
 import { Config, defaultConfig } from '../model/config';
 import { TimeRange } from '../utils/datetime';
 
 const host = '/api/proxy/plugin/network-observability-plugin/backend/';
 
-export const getFlows = (params: FlowQuery): Promise<Record[]> => {
+export const getFlows = (params: FlowQuery): Promise<RecordsResult> => {
   return axios.get(host + '/api/loki/flows', { params }).then(r => {
     if (r.status >= 400) {
       throw new Error(`${r.statusText} [code=${r.status}]`);
     }
-    return (r.data.data.result as StreamResult[]).flatMap(r => parseStream(r));
+    const aggQR: AggregatedQueryResponse = r.data;
+    return {
+      records: (aggQR.result as StreamResult[]).flatMap(r => parseStream(r)),
+      stats: aggQR.stats
+    };
```

Comment:
Need to do the same logic in `getTopology`

Reply:
`return (r.data.data.result as TopologyMetrics[])` needs to be `return (r.data.result as TopologyMetrics[])`